scl/paloalto/panos.conf

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+# Copyright (c) 2020 Balabit
+# created by mileek@gmail.com
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
```

Comment:
So that this could be installed and packaged: please edit `CMakeLists.txt` and `Makefile.am` in the scl directory.

Reply:
OK

Reply:
Not sure what I need to edit in `Makefile.am`

Reply:
You need to add the directory to `SCL_SUBDIRS`, similarly to the other scl-s.

That variable will be later used to fill the `EXTRA_DIST` variable, which will make sure these directories end up in the tarball.

Reply:
I added dir to `CMakeLists.txt` but not sure what to do edit in `Makefile.am` as you mentioned.

Reply:
Under https://github.com/syslog-ng/syslog-ng/blob/7f21db29622c67a00e514cfa45b3d38bfa884e15/scl/Makefile.am#L24
just add `paloalto \`